packages/cli/docs-resolver/src/DocsDefinitionResolver.ts

```diff
@@ -160,18 +180,23 @@ export class DocsDefinitionResolver {
 
         // postprocess markdown files after uploading all images to replace the image paths in the markdown files with the fileIDs
         const basePath = this.getDocsBasePath();
+
+        // TODO: include more (canonical) slugs from the navigation tree
+        const markdownFilesToPathName: Map<AbsoluteFilePath, string> = new Map();
```

Comment:
nit: typically the rest of the codebase will use `Record` since it is more typesafe than a map

Defer to you what you think is better here (sometimes there are better reasons to use a map like preserving order of insertion)